mayhem/temp.cpp

```diff
@@ -0,0 +1,13 @@
+#include <stdint.h>
```

Comment:
This fuzz target doesn't fuzz anything, is that intentional?

Reply:
Oops, no it isn't. Removed the file